Resources/Prototypes/Entities/Mobs/Player/familiars.yml

```diff
@@ -53,7 +56,7 @@
   - type: MobMover
   - type: Faction
     factions:
-    - SimpleNeutral
+    - PetsNT
```

Comment:
Cerberus isn't an NT pet, it's an antag pet.

Reply:
You mean give him back a neutral faction? The AI should not attack players on his own.